utils.py

```diff
@@ -1022,30 +1022,40 @@ def get_fake_context(context=None, area_type='VIEW_3D'):
 
 
 def has_url(text):
-    # urls = re.findall('https?://(?:[-\w.]|(?:%[\da-fA-F]{2}))+', text)
-    urls = re.findall(r'(?<=<a href=")[^"]*', text)
-    urls_with_str = []
+    #first remove markdown *
+    text = text.replace('*','')
```

Comment:
So the markdown will not be displayed in add-on? Is one of the functions of has_url to remove markdown syntax?

Reply:
yeah, no bold text or other in Blender UI code.